docs/Dialects/onnx.md

```diff
@@ -1333,15 +1333,15 @@ Effects: MemoryEffects::Effect{}
 
 | Operand | Description |
 | :-----: | ----------- |
-| `X` | tensor of 16-bit float values or tensor of 32-bit float values or tensor of 64-bit float values
-| `W` | tensor of 16-bit float values or tensor of 32-bit float values or tensor of 64-bit float values
+| `X` | tensor of 16-bit float values or tensor of 32-bit float values or tensor of 64-bit float values or unranked.tensor of 32-bit float values or tensor of 32-bit float values with layout NCHWxCand factors 4, 0
+| `W` | tensor of 16-bit float values or tensor of 32-bit float values or tensor of 64-bit float values or unranked.tensor of 32-bit float values or tensor of 32-bit float values with layout KCNMxCyKand factors 4, 4
```

Comment:
I am curious how these info were generated for Conv or they were carried out from the old PR.

Reply:
hmm, good question. Let me look into this.

Reply:
fixed